.cirrus.yml

```diff
@@ -27,24 +27,23 @@ env:
     #### Cache-image names to test with (double-quotes around names are critical)
     #### Comment out fedora-35 for podman 4.x branches.
     ####
-    FEDORA_NAME: "fedora-36"
+    FEDORA_NAME: "fedora-37"
     FEDORA_AARCH64_NAME: "${FEDORA_NAME}-aarch64"
-    #PRIOR_FEDORA_NAME: "fedora-35"
+    PRIOR_FEDORA_NAME: "fedora-36"
     UBUNTU_NAME: "ubuntu-2204"
 
     # Image identifiers
-    IMAGE_SUFFIX: "c4678746211876864"
+    IMAGE_SUFFIX: "c5178639502278656"
     # EC2 images
     FEDORA_AMI: "fedora-aws-${IMAGE_SUFFIX}"
     FEDORA_AARCH64_AMI: "fedora-podman-aws-arm64-${IMAGE_SUFFIX}"
     # GCP Images
     FEDORA_CACHE_IMAGE_NAME: "fedora-${IMAGE_SUFFIX}"
-    #PRIOR_FEDORA_CACHE_IMAGE_NAME: "prior-fedora-${IMAGE_SUFFIX}"
-    UBUNTU_CACHE_IMAGE_NAME: "ubuntu-${IMAGE_SUFFIX}"
+    PRIOR_FEDORA_CACHE_IMAGE_NAME: "prior-fedora-${IMAGE_SUFFIX}"
+    #UBUNTU_CACHE_IMAGE_NAME: "ubuntu-${IMAGE_SUFFIX}"
```

Comment:
There's a hecka lot of commented-out stuff here. Some day someone will need to add them back, and will miss one even if they do a `git log -1 -p` of this commit to make sure.

Could you add `# FIXME: reenable once <BUGID> is resolved` to each commented-out section?

Reply:
Yep I can/should add a summary comment.  No problem.  I thought I made this a separate commit so it can just be reverted, no?

I'm hoping to get the debian stuff rolling before too long, so hopefully that can replace Ubuntu entirely.

Reply:
`revert` assumes an unchanging `.cirrus.yml`, and that will be void within (I hope) one day of this PR merging.